--- ./xxd.c	2023-06-13 22:11:11.553091188 -0700
+++ ./xxd.c	2023-06-13 22:35:35.417880872 -0700
@@ -141,7 +141,7 @@
 extern long int strtol();
 extern long int ftell();
 
-char version[] = "xxd V1.10 27oct98 by Juergen Weigert";
+char version[] = "xxd V1.10 25july23 by Juergen Weigert";
 #ifdef WIN32
 char osver[] = " (Win32)";
 #else
@@ -215,7 +215,7 @@
 static void xxdline __P((FILE *, char *, int));
 
 #define TRY_SEEK	/* attempt to use lseek, or skip forward by reading */
-#define COLS 256	/* change here, if you ever need more columns */
+#define COLS 1024	/* change here, if you ever need more columns */
 #define LLEN (11 + (9*COLS-1)/1 + COLS + 2)
 
 char hexxa[] = "0123456789abcdef0123456789ABCDEF", *hexx = hexxa;
